src/datadoc/backend/core.py

```diff
@@ -75,8 +88,10 @@ def __init__(
             dataset_path (Optional): The file path to the dataset.
             metadata_document_path (Optional): The file path to the metadata document.
             statistic_subject_mapping (Optional): An instance of StatisticSubjectMapping.
+            errors_as_warnings (Optional): Disable raising exceptions if inconsistencies are found between existing and extracted metadata.
```

Comment:
Based on the google style guide I have now removed types from all docstrings if they are provided in code